src/modules/jcms_rest/src/Routing/RouteSubscriber.php

```diff
@@ -18,7 +18,11 @@ class RouteSubscriber extends RouteSubscriberBase {
    * {@inheritdoc}
    */
   public static function getSubscribedEvents() {
-    return [RoutingEvents::ALTER => ['onAlterRoutes', -9999]];
+    $events = parent::getSubscribedEvents();
+
+    $events[RoutingEvents::ALTER] = ['onAlterRoutes', -9999];
+
+    return $events;
```

Comment:
This was not needed but it seemed to be the incorrect approach to disregard other subscribed events.